### DISCUSSION TITLE
[MultiSwitch]: Add Icon prop to MultiSwitch component items

### DISCUSSION THREAD

**Discussion by AlexSKuznetsov** [epam/UUI#768] [Q&A] [answered]
## Description
Sometimes we need to have a MultiSwitch button not with the Caption text, but just with the icons. (UUI3 Loveship)
& provide cx for styling

<img width="96" alt="Screenshot 2022-10-13 at 11 39 14" src="https://user-images.githubusercontent.com/47721260/195547010-844984ab-542f-4fe6-a712-aab79477b8cb.png">

## Describe the solution you'd like

```js
import IconOne from '.../assets';
import IconTwo from '.../assets';


<MultiSwitch
    onValueChange={(newValue) => { ... }}
    items={[{"id":1,"icon": IconOne},{"id":2,"icon": IconTwo}]}
    value={1}
    cx={someClass}
/>
```

**Comment by AlekseyManetov** [comment, marked as answer]
Hi, you already can make MultiSwitch with icons - https://codesandbox.io/s/uui-forked-9xpf10?file=/Example.tsx

**Comment by AlexSKuznetsov** [comment]
Hi, cool, thank you. I think I will close this
